tests/sentry/event_manager/test_severity.py

```diff
@@ -92,7 +92,7 @@ def test_message_event_simple(
             {"logentry": {"message": "Dogs are great!"}},
         ]
         for case in cases:
-            manager = EventManager(make_event(**case))
+            manager = EventManager(make_event(level="error", **case))
```

Comment:
nit: iirc `make_event` sets `level` to "error" by default, is there a reason this is made explicit here?